_component_design/time-selector.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Time Selector
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: The time selector enables the user to select a time from a list of values. The time selector is often accompanied by a date selector that a user can use to select a date.
+preview-image: preview-images/time-selector.svg
+resource: true
+status: stable
+last-modified: 2018-11-07
+helix-ui-css: false
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use the time selector when you want to enable a user to select a time from a list of values. The time selected includes an hour value and a minute value. The hour value can include 12-hour clock values, or 24-hour clock values. Minutes are grouped into 10-minute, 15-minute, or 30-minute intervals. For example, a user can select a time of **1:00 PM**, **1:10 PM**, **1:15 PM**, or **1:30 PM**, but cannot select any other minute values.
+
+If you want the user to select a time from a limited list of predetermined times, consider using [radios]({{site.baseurl}}\components\radios.html).
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
```

Comment:
add a link to text conventions dat & time for best practices in time display

Reply:
Yeppers, another good place for a xref link. Done, son!